src/modules/partners/partner_test.ts

```diff
@@ -40,48 +41,42 @@ const expectBody = expect.objectContaining({
       total_village: expect.any(Number),
       logo: expect.any(String),
       created_at: expect.any(String),
-      website: expect.any(String)
-    })
+      website: expect.any(String),
+    }),
   ]),
-  meta: expectMeta
+  meta: expectMeta,
 })
 
 describe('tests partners', () => {
-  it('test success findAll', async () => {
-    return request(app)
-      .get('/v1/partners')
-      .expect(200)
-      .then((response) => {
-        expect(response.body).toEqual(expectBody)
-      })
-  })
+  it('test success findAll', async () => request(app)
+    .get('/v1/partners')
+    .expect(200)
+    .then((response) => {
+      expect(response.body).toEqual(expectBody)
+    }))
```

Comment:
Opinion: I think it's easier to read the old version :/

Reply:
but the extended airbnb is already very good compared to the standard one